modules/codemod/lib/v5/maps.ts

```diff
@@ -127,7 +127,6 @@ export const bundleExportMap: {
   accessibleHide: '@workday/canvas-kit-react/common',
   getErrorColors: '@workday/canvas-kit-react/common',
   errorRing: '@workday/canvas-kit-react/common',
-  memoizedFocusRing: '@workday/canvas-kit-react/common',
```

Comment:
Removed this now import is gone

Reply:
We don't modify old codemods. If someone is upgrading from v4, this codemod would still be necessary.

Reply:
Whoops my bad, didnt see it was a codemod, reverting now.